RELEASE/scripts/autoscend/combat/auto_combat_default_stage2.ash

```diff
@@ -138,6 +138,31 @@ string auto_combatDefaultStage2(int round, monster enemy, string text)
 		}
 	}
 
+	if(!combat_status_check("banishercheck") && auto_wantToBanish(monster_phylum(enemy), my_location()))
+	{
+		string banishAction = banisherCombatString(monster_phylum(enemy), my_location(), true);
+		if(banishAction != "")
+		{
+			auto_log_info("Looking at banishAction: " + banishAction, "green");
+			combat_status_add("banisher");
+			if(index_of(banishAction, "skill") == 0)
+			{
+				handleTracker(monster_phylum(enemy), to_skill(substring(banishAction, 6)), "auto_banishes");
+			}
+			else if(index_of(banishAction, "item") == 0)
+			{
+				handleTracker(monster_phylum(enemy), to_item(substring(banishAction, 5)), "auto_banishes");
+			}
+			else
+			{
+				auto_log_warning("Unable to track banisher behavior: " + banishAction, "red");
+			}
+			return banishAction;
+		}
+		//we wanted to banish an enemy and failed. set a property so we do not bother trying in subsequent rounds
+		combat_status_add("banishercheck");
```

Comment:
I think this banisher check should by phylum specific, like `phylumbanishercheck` or similar

Concern, which I haven't tested/looked at a log, is if we fail to get a phylum banish, will this prevent it from getting a normal banisher?

Reply:
I suspect that it would prevent it because of the combat status check so updated to phylumbanishercheck